src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/Handlers/AbstractEvaluationCommandLineHandlerTests.ConcreteAbstractEvaluationCommandLineHandler.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Collections.ObjectModel;
+using Microsoft.VisualStudio.ProjectSystem.Logging;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices.Handlers
+{
+    partial class AbstractEvaluationCommandLineHandlerTests
+    {
+        private class ConcreteAbstractEvaluationCommandLineHandler : AbstractEvaluationCommandLineHandler
```

Comment:
Worst name ever?

Reply:
Ha, that made me lol - didn't think about the name just appended concrete to original name without thinking. Still getting used to the "Abstract" naming which came from Roslyn.

Reply:
Fixed.